src/test/regress/expected/limit_intermediate_size.out

```diff
@@ -39,42 +39,33 @@ ERROR:  the intermediate result size exceeds citus.max_intermediate_result_size
 DETAIL:  Citus restricts the size of intermediate results of complex subqueries and CTEs to avoid accidentally pulling large result sets into once place.
 HINT:  To run the current query, set citus.max_intermediate_result_size to a higher value or -1 to disable.
 -- router queries should be able to get limitted too
-SET citus.max_intermediate_result_size TO 3;
+SET citus.max_intermediate_result_size TO 2;
```

Comment:
Note that the query before the change was pulling all the data to together in the last step (subquery `a`). So, with this commit I made the test comply with the comment. Now, each cte is small, the top level query is only on the aggregates of the results.